docs/admin/install/prepare.rst

```diff
@@ -38,15 +42,21 @@ If the Qubes hardware compatibility list entry for your computer recommends the
 
 Download and verify Qubes OS
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-On the working computer, download the Qubes OS ISO for version ``4.1.2`` from `https://www.qubes-os.org/downloads/ <https://www.qubes-os.org/downloads/#qubes-release-4-1-2>`_. The ISO is 5.4 GiB approximately, and may take some time to download based on the speed of your Internet connection.
+On the working computer, download the Qubes OS ISO and cryptographic hash values for version ``4.2.2rc1`` from `https://www.qubes-os.org/downloads/ <https://www.qubes-os.org/downloads/#qubes-release-4-2-2-rc1>`_. The ISO is 6.9 GB approximately, and may take some time to download based on the speed of your Internet connection.
+
+Follow the linked instructions to `verify the ISO <https://www.qubes-os.org/security/verifying-signatures/#how-to-verify-detached-pgp-signatures-on-qubes-isos>`_. If both the ISO and cryptographic hash values are stored in the same directory, you can quickly verify by running:
```

Comment:
Instead of "If both ...are stored in the same directory" can we tell them to download them to the same directory?

"Ensure that the ISO and hash values are in the same directory, then run ..." ?